ocaml/idl/datamodel.ml

```diff
@@ -6625,7 +6637,7 @@ let vdi_get_nbd_info = call
     ~errs: [Api_errors.vdi_incompatible_type]
     ~result:(Set String, "The list of URIs.")
     ~doc:"Get a list of URIs specifying how to access this VDI via the NBD server of XenServer. A URI will be returned for each PIF of each host that is connected to the VDI's SR. An empty list is returned in case no network has a PIF on a host with access to the relevant SR. To access the given VDI, any of the returned URIs can be passed as the export name to the nbd-server running at the IP address and port specified by that URI."
-    ~allowed_roles:_R_VM_ADMIN
```

Comment:
This doesn't seem right. Why so restrictive? This should be callable by backup software.

Reply:
oops good spot, meant to change it for set_cbt_enabled

Reply:
It has the same RBAC role that is required for connecting to the NBD server, VM Administrator.